test/support/reader.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Membrane.Opus.Support.Reader do
+  def read_packets(filename) do
+    filename
+    |> File.read!()
+    |> String.split()
+    |> Enum.map(fn x ->
+      {:ok, y} = Base.decode16(x)
+      y
+    end)
```

Comment:
I used almost the same code in plugin rtp fixtures. Maybe this could become part of either `Bunch` or `Membrane.Testing`?

Reply:
IMHO it's much better to do `:erlang.binary_to_term` instead. Left that just because it was already done.

Reply:
`erlang.binary_to_term` has no use here as the binary files don't come from Erlang VM but from Wireshark

Reply:
ok, so I'd rather see it in bunch